packages/lesswrong/lib/collections/tags/subforumHelpers.ts

```diff
@@ -22,3 +22,8 @@ export const subforumSortingToResolverName = (sort: SubforumSorting) =>
 
 export const isSubforumSorting = (sort: string): sort is SubforumSorting =>
   (subforumSortings as string []).includes(sort);
+
+export const subforumLayouts = ["feed", "list"] as const
```

Comment:
You can prevent the below ugly cast by using the purpose-built TupleSet.

```suggestion
export const subforumLayouts = new TupleSet(["feed", "list"] as const)
```